test/presenters/parts_test.rb

```diff
@@ -125,6 +125,21 @@ def requested_content_item_path
     refute @parts.has_valid_part?
   end
 
+  test 'does not request a part when valid url of first part is manually entered into the address bar' do
```

Comment:
The test title does not quite match the implementation as it does "request a part", it isn't a valid part.